source/administrator-guide/managing-users-and-groups/authentication-mode.rst

```diff
@@ -13,17 +13,308 @@ Configuring LDAP
 ----------------
 
 To enable LDAP, setup authentication by including ``WEB-INF/config-security/config-security-ldap.xml``
-in ``WEB-INF/config-security/config-security.xml``. You can then configure your environment in ``config-security-ldap-overrides.properties``.
+in ``WEB-INF/config-security/config-security.xml``, uncommenting the following line:
 
+.. code-block:: xml
+
+    <import resource="config-security-ldap.xml"/>
+
+The LDAP configuration is defined in ``WEB-INF/config-security/config-security.properties``, you can then configure
+your environment updating the previous file or overriding the properties in the file
+``WEB-INF/config-security/config-security-overrides.properties``.
+
+1. Define the LDAP connection:
+
+    - ``ldap.base.provider.url``: This tells the portal where the LDAP server is located. Make sure that the computer with the catalog can hit the computer with the LDAP server. Check to make sure that the appropriate ports are opened, etc.
+
+    - ``ldap.base.dn``: this will usually look something like: “dc=organizationnamehere,dc=org”
+
+    - ``ldap.security.principal`` & ``ldap.security.credentials``: Define LDAP administrator user to use to bind to LDAP. If not define, an anonymous bind is made. Principal is the username and credentials property the password.
+
+    .. code-block:: text
+
+        # LDAP security properties
+        ldap.base.provider.url=ldap://localhost:389
+        ldap.base.dn=dc=fao,dc=org
+        ldap.security.principal=cn=admin,dc=fao,dc=org
+        ldap.security.credentials=ldap
+
+
+    To verify that you have the correct settings, try to connect to the LDAP server using an LDAP browser application.
+
+2. Define where to find users in LDAP structure for authentication:
+
+    - ``ldap.base.search.base``: this is where the catalog will look for users for authentication.
+
+    - ``ldap.base.dn.pattern``: this is the distinguished name for the user to bind with. {0} is replaced by the user name typed in the sign in screen.
+
+    .. code-block:: text
+
+        ldap.base.search.base=ou=people
+        ldap.base.dn.pattern=uid={0},${ldap.base.search.base}
+        #ldap.base.dn.pattern=mail={0},${ldap.base.search.base}
+
+
+Authorization Settings
+======================
+
+When using LDAP, the user information and privileges for GeoNetwork could be defined from the LDAP attributes.
+
+User information
+````````````````
+
+The user information could be retrieved from the LDAP configuring for each user attribute in the catalog database which LDAP attributes match.
+If the attribute is empty or not defined, a default value could be defined. The configuration is the following:
```

Comment:
A more explicit explanation about how to set the default value:
The property value is compose by two parts separated by ``,`` character. The first part is the attribute name and the second part is the default value in case the attribute name is not define or the attribute value in LDAP is empty.